### PR TITLE
update filter for e2e-aws-ipi-disconnected-private-sdn-workers-rhcos-rhel8-p2

### PR DESCRIPTION
**Pull request by Xia-Zhao-rh**

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml [hunk 1]
```diff
@@ -353,6 +353,7 @@ tests:
       E2E_RUN_TAGS: '@aws-ipi and @disconnected and @network-openshiftsdn and not
         @fips'
       TAG_VERSION: '@4.12'
+      TEST_FILTERS: ~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&
```
Comment (kuiwang02) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml:356]: @Xia-Zhao-rh better be `~ChkUpgrade&;~NonPreRelease&;~Serial&;~Disruptive&;~ConnectedOnly&;~HyperShiftMGMT&`

Comment (Xia-Zhao-rh) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml]: Do we need to ~HyperShiftMGMT& to other profile?

Comment (kuiwang02) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml]: for connected, it is default to added
for existing disconnected, better to be. at that time, we have no this label.

**Pull request merged**